### PR TITLE
Fixes #35146 - Lock rack-test < 2.0.0

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

**Comment by stejskalleos** [comment]
Changes LGTM, but do we have redmine issue for tracking this problem? Sooner or later we would need to fix it, would be nice to track it.

**Comment by ofedoren** [comment]
Thanks, @stejskalleos, here is an issue: https://projects.theforeman.org/issues/35150

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

**Comment by stejskalleos** [comment]
Thanks @ofedoren!

**Comment by ekohl** [comment]
Is this to resolve the `cookie_jar` error we saw? If so, mentioning that in the commit message can help others find this.

File: bundler.d/test.rb [hunk 1]
```diff
@@ -5,8 +5,11 @@
   gem 'minitest-reporters', '~> 1.4', :require => false
   gem 'minitest-retry', '~> 0.0', :require => false
   gem 'minitest-spec-rails', '~> 6.0'
+  # Lock rack-test until some of below deps are fixed
+  gem 'rack-test', '< 2'
```
Comment (ekohl) [inline bundler.d/test.rb:9]: Note this is actually a runtime dependency (actionpack pulls it in). I wonder if the pin should live at the top level. I'll dive further into this.

Comment (ekohl) [reply bundler.d/test.rb:9]: Looks like it's `show_me_the_cookies`: https://github.com/nruth/show_me_the_cookies/pull/64. https://github.com/theforeman/foreman/pull/9353

**Pull request merged**